app/Database/Requirement.hs

```diff
@@ -0,0 +1,39 @@
+{-|
+Description: Requirement
+
+Module containing data type that represents a "Requirement".
+
+We will use parsed data to create instances of this type.
+-}
+module Requirement
+( ProgramReq(..)
+, CourseReq(..)
+, Req(..)
+) where
+
+import qualified Data.String as S
+
+
+data ProgramReq = PRGREQ String [Req]
+
+-- | Returns a well formatted String representing a program requirement for specified program.
+instance Show ProgramReq where
+	show (PRGREQ program reqs) = "Program Requirements for " ++ program ++ ":\n" ++ map show reqs
```

Comment:
It seems you're using tabs instead of spaces; make sure to use exactly 4 spaces for indentation.